tests/test_bzl_rules.cpp

```diff
@@ -0,0 +1,65 @@
+#include "flatbuffers/flexbuffers.h"
```

Comment:
this file is weird to me. It doesn't actually test anything, and has a bunch of unused code. Can it be cleaned up? removed? replaced with just using test.cpp?